CoroutineTemplate/domain/src/main/java/co/nimblehq/coroutine/entity/UserEntity.kt

```diff
@@ -0,0 +1,26 @@
+package co.nimblehq.domain.entity
+
+data class UserEntity(
+    val id: Int?,
+    val name: String,
+    val username: String,
+    val email: String?,
+    val address: Address?,
+    val phone: String,
+    val website: String
+) {
+
+    data class Address(
+        val street: String,
+        val suite: String,
+        val city: String,
+        val zipCode: String,
+        val geo: Geo?
+    ) {
+
```

Comment:
Same here: Can we get rid of this line break in case we don't have the `@Parcelize` annotation to be consistent with the `UserResponse`?